src/sidebar/templates/annotation.html

```diff
@@ -8,8 +8,13 @@
     <span ng-if="vm.user()">
       <a class="annotation-header__user"
         target="_blank"
+        ng-if="vm.isHAuthority()"
         ng-href="{{vm.serviceUrl('user',{user:vm.user()})}}"
         >{{vm.username()}}</a>
+      <a class="annotation-header__user"
```

Comment:
If the username is not a link we should just make it plain text (ie. a `<span>`) to avoid confusing screen readers or inheriting any link-specific behaviour from the browser.